pkg/virtctl/convert/guess.go

```diff
@@ -0,0 +1,36 @@
+package convert
+
+import (
+	"bufio"
+	"bytes"
+	"io"
+	"k8s.io/client-go/pkg/util/yaml"
+	"unicode"
+)
+
+func GuessStreamType(source io.Reader, size int) (io.Reader, Type) {
```

Comment:
My assumption was, that the conversion later on will fail, if we have an unsupported function. I will think about it a little bit more.